File: src/lib/types/radiology.ts
export interface Radiology {
  id: number;
  patient_id: number;
  x_ray: string;
  posture_frontend: string;
  posture_backend: string;
  posture_left: string;
  posture_right: string;
  created_at: number;
}

export const DefaultRadiology: Radiology = {
  id: 0,
  patient_id: 0,
  x_ray: "",
  posture_frontend: "",
  posture_backend: "",
  posture_left: "",
  posture_right: "",
  created_at: 0,
};

File: src-tauri/src/schema/radiology.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Serialize, Deserialize)]
pub struct Radiology {
    #[serde(default)]
    pub id: i64,

    pub patient_id: i64,
    pub x_ray: String,
    pub posture_frontend: String,
    pub posture_backend: String,
    pub posture_left: String,
    pub posture_right: String,
    pub created_at: i64,
}
